.github/workflows/ci.yml

```diff
@@ -94,3 +94,9 @@ jobs:
          mkdir t
          PATH=/usr/local/nginx/sbin:$PATH \
          prove -v -Itest-nginx/lib cases/
+     - name: nginx test
+       working-directory: tests/nginx-tests
+       env:
+         TEST_NGINX_BINARY: /usr/local/nginx/sbin/nginx
+       run: |
+         sudo TEST_NGINX_BINARY=/usr/local/nginx/sbin/nginx prove -Inginx-tests/lib $(ls -d nginx-tests/* | grep -v http_ssl_asynchronous_mode)
```

Comment:
hi, remove changes for ci.yml, let me merge ****.t file firstly!

Reply:
And later we can post an new nginx-tests.yml for these tests in nginx-tests/nginx-tests

Reply:
ok